.travis.yml

```diff
@@ -31,8 +31,6 @@ addons:
   postgresql: "9.5"
 node_js:
   - "12"
-  - "10"
```

Comment:
So do these changes remove Node 10 support?

Reply:
Nope just removed so travis finishes faster, should work on 12,10,8